### DISCUSSION TITLE
why my github code block background color is blue permanently.

### DISCUSSION THREAD

**Discussion by mukulietlucknow** [community/community#52885] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

![image](https://user-images.githubusercontent.com/25330845/232190626-9beda65e-3e03-4aa0-a986-5b8e65e71e5a.png)

I have checked all the settings but in my all repo this background color is not going it is only blue and it makes readability very bad

can someone help me regarding it

**Comment by CloudClown** [comment]
The same problem is happening to me exclusively on Chorme. And it appears properly in another browser.

**Reply by iRakic** [reply]
I have the same issue. I'm using Brave as a browser which is Chromium based.

**Comment by NathanHam16** [comment]
Maybe clear cache / history on browser?

**Reply by mukulietlucknow** [reply]
I have tried but it is not useful

**Comment by Luckshya1210** [comment]
Same issue😥

**Comment by Nehag7** [comment]
can anyone tell me how we can fix it on Chrome or send this bug to the Git team. so that they can fix it.

**Comment by cfry** [comment]
Same problem on MAcOS version 13.2.1, Chrome Version 111.0.5563.146
Just showed up in the last couple weeks or so.

**Comment by mtayyab03** [comment]
same issue is coming on my github,please any one can help

**Comment by gutche** [comment]
Valve pls fix

**Comment by sumit-kumar-yadav** [comment]
Same issue. How to fix that?

**Comment by HappyGick** [comment]
Same issue here.

**Comment by queenofcorgis** [comment]
Hey folks - Lili, one of the community managers here 👋🏼  We've opened an issue for this internally and I'll keep you posted with any updates. Thanks for reporting this 👍🏼

**Comment by iRakic** [comment]
It looks like it's fixed now.
Update: Actually after checking it again later there is still an issue. In the new code view, it looks okay.

**Reply by HappyGick** [reply]
Not for me. I'm using Brave (Chromium based).

**Reply by HappyGick** [reply]
Using the New Code View feature preview helps (makes it so the background no longer looks blue). But the old one is still bugged.

**Comment by PreetGarach** [comment]
I was facing the same issue. Try removing third party extensions in chrome. It worked for me!!

**Reply by mbechev** [reply]
Correct! 

In my case, disabling the [Blackbox Code Search Autocomplete Extraction](https://chrome.google.com/webstore/detail/blackbox-code-search-auto/mcgbeeipkmelnpldkobichboakdfaeon) extension fixed the problem. 🚀

**Reply by ghost** [reply]
thanks this worked.

**Reply by iRakic** [reply]
> Try removing third party extensions in chrome. It worked for me!!

That was the problem for me as well

**Reply by urosctd** [reply]
Same here

**Reply by nayanp35** [reply]
Life saver
